packages/server/lib/controllers/files.js

```diff
@@ -108,7 +108,8 @@ module.exports = {
 
     debug('test url for file %o', { file, url })
 
-    return url
+    // https://github.com/cypress-io/cypress/issues/5909
+    return url.replace(/\+/g, '%2B')
```

Comment:
Let's put this along with the other character replacements in [escape_filename.ts](https://github.com/laistomazz/cypress/blob/issue-5909/packages/server/lib/util/escape_filename.ts).

Reply:
Done. I'm not sure the way I added the comment follows your coding style though, can you please check it?